server/models/subscription.ts

```diff
@@ -206,9 +213,11 @@ export default (
           notification.id
         }`;
       }
-      if (msg && subscription.immediate_email) sendImmediateNotificationEmail(subscription, msg);
-      return nRead;
-    }));
+      if (msg && nRead.Subscription.immediate_email) {
```

Comment:
add a ? to Subscription here. or we need to require the Subscription model above in the query.

Reply:
It should always be present, so I can require it as a formality, but shouldn't change the result.

Reply:
I worry about Subscriptions that were deleted but notifications that were improperly deleted from the cascade, but I guess that's not really the kind of issue we should be worried about here.